Makefile

```diff
@@ -241,6 +241,11 @@ doc: install-doc
 	rm ./pachctl
 	mv pachctl.rst doc/pachctl
 
+launch-monitoring:
+	kubectl create -f ./etc/plugin/monitoring
+	sleep 120
```

Comment:
Sleeping for stuff like this always makes me a bit nervous. I actually made the `check_pachd_ready.sh` script generic in another PR that was never merged. You could cherry pick the commit: 98958587bc3082a8e3b0c227ee60a5c90d2dee80 and it should allow you to use the script instead of the sleep.